repo/packages/S/snaplogic-snaplex/0/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "packagingVersion": "2.0",
+    "description": "Snaplogic Snaplex",
+    "maintainer": "support@snaplogic.com",
+    "framework": false,
+    "name": "snaplogic-snaplex",
+    "postInstallNotes": "Snaplogic Snaplex has been installed.",
+    "postUninstallNotes": "Snaplogic Snaplex has been uninstalled. Note that any data persisted to a NFS share still exists and will need to be manually removed.",
+    "selected": true,
+    "tags": [
+      "data-integration",
+      "JCC",
+      "Snaplogic"
+    ],
+    "version": "latest",
```

Comment:
please use version of the format: integrationVersion-baseTechVersion. Ex: 0.1.0-1.2.3 if 1.2.3 is Snaplogic version